Rules/CommonScripts/PlayerCamera.as

```diff
@@ -197,11 +203,94 @@ void onRender(CRules@ this)
 		ul -= size * 0.5;
 		lr = ul + size;
 		//wiggle up and down
-		f32 wave = Maths::Sin(getGameTime() / 10.0f) * 5.0f;
+		float wave = Maths::Sin(getGameTime() / 10.0f) * 5.0f;
 		ul.y += wave;
 		lr.y += wave;
 		//draw
 		GUI::DrawButtonPressed(ul - Vec2f(10, 10), lr + Vec2f(10, 10));
 		GUI::DrawText(text, ul, SColor(0xffffffff));
 	}
 }
+
+void onTick(CRules@ this)
+{
+	if (isCinematic())
```

Comment:
this is opening and reading a file every tick, we shouldn't be doing that. We should only read the file once. And only write to it on changes.